stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -588,16 +613,25 @@ private class TFEState {
       debugLog("Creating a new op based on type \(opType).")
       let op: CTFEOp? = TFE_NewOp(context.eagerContext, opType, status)
       checkOk(status)
+      var deviceName: String
```

Comment:
This need not be `var` because both assignments (in `if` and `else` branches) are initialization, not mutation.

```suggestion
      let deviceName: String
```